core/src/main/java/org/mule/runtime/core/api/rx/Exceptions.java

```diff
@@ -153,4 +155,29 @@ public static MuleException rxExceptionToMuleException(Throwable throwable)
     }
   }
 
+  /**
+   * Propagate an exception through streams even if it's considered fatal by Reactor.
+   *
+   * @param t throwable.
+   * @return exception wrapped by Reactor, after possibly wrapping with {@link MuleFatalException}.
+   */
+  public static RuntimeException propagateFatal(Throwable t) {
```

Comment:
Maybe `propagateWrapFatal` `propagateWrappingFatal` then? Just because in most cases this will not actually propagate a fatal